packages/contracts/src/dollar/core/CreditClock.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.3;
+
+import "../libs/ABDKMathQuad.sol";
+import "./UbiquityDollarManager.sol";
+
+contract CreditClock {
+    using ABDKMathQuad for uint256;
+    using ABDKMathQuad for bytes16;
+
+    // Manager contract.
+    UbiquityDollarManager private immutable manager;
+
+    // ABDKMathQuad with value of 1.
+    bytes16 private immutable one = uint256(1).fromUInt();
+
+    // The block height from where we start applying the rate.
+    uint256 public rateStartBlock;
+
+    // This is the exchange rate of uAR for the start block.
```

Comment:
```suggestion
    // This is the exchange rate of Ubiquity Credits for the start block.
```

Reply:
We do not allow for the use of token symbol references in our code because we have the ability to update them, as well as to enhance clarity for new developers.